packages/frontend2/src/app/(legacy)/scaling-next/projects/[slug]/_components/value-locked-summary.tsx

```diff
@@ -0,0 +1,197 @@
+import { type WarningWithSentiment } from '@l2beat/config'
+import { UpcomingBadge } from '~/app/_components/badge/upcoming-badge'
+import { ValueLockedBreakdown } from '~/app/_components/breakdown/value-locked-breakdown'
+import { PercentChange } from '~/app/_components/percent-change'
+import { Square } from '~/app/_components/square'
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from '~/app/_components/tooltip/tooltip'
+import { RoundedWarningIcon } from '~/icons/rounded-warning'
+import { type ScalingProjectEntry } from '~/server/features/scaling/project/get-scaling-project-entry'
+import { formatCurrency } from '~/utils/format'
+import { unifyPercentagesAsIntegers } from '~/utils/math'
+
+export interface ValueLockedBreakdown {
+  totalChange: number
+  total: number
+  canonical: number
+  external: number
+  native: number
+  warning: WarningWithSentiment | undefined
+}
+
+export interface ValueLockedSummaryProps {
+  breakdown:
+    | NonNullable<ScalingProjectEntry['header']['tvl']>['tvlBreakdown']
+    | undefined
+  isArchived?: boolean
+}
+
+// TODO (scaling-project-page): Add tvl breakdown link
```

Comment:
Shouldn't we add it now? The link will not change.

Reply:
yup